netlify/build

```diff
@@ -14,6 +14,7 @@ popd
 gem install bundler
 bundle install
 build _config_cockroachdb.yml
+exit $?
```

Comment:
I don't know bash very well, but why are we handling this differently than for htmltest?

```
if [[ $? != 0 ]]; then
  exit 1
fi;
```